ctapipe/tools/process.py

```diff
@@ -316,7 +320,6 @@ def finish(self):
         Last steps after processing events.
         """
         self.write.write_simulation_histograms(self.event_source)
-        self.write.finish()
         self.event_source.close()
```

Comment:
wouldn't this be handled by the exit stack of event source?

Reply:
yes